databricks/koalas/frame.py

```diff
@@ -4344,24 +4344,6 @@ def drop(self, labels=None, axis=1,
         0  1  7
         1  2  8
 
-        >>> pdf = pd.DataFrame({'x': [1, 2], 'y': [3, 4], 'z': [5, 6], 'w': [7, 8]})
```

Comment:
Let's have this doctests.

Need to add `columns=[...]` for py3.5.

```py
>>> df = ks.DataFrame({'x': [1, 2], 'y': [3, 4], 'z': [5, 6], 'w': [7, 8]}, columns=['x', 'y', 'z', 'w'])
```

or we can do:

```py
>>> df = ks.DataFrame({('a', 'x'): [1, 2], ('a', 'y'): [3, 4], ('b', 'z'): [5, 6], ('b', 'w'): [7, 8]},
...                   columns=[('a', 'x'), ('a', 'y'), ('b', 'z'), ('b', 'w')])
```

Reply:
@ueshin  Really appreciate your comment Takuya!! I was really suffered all last night from it had never passed lint-test 😭

Reply:
@itholic 
In python < 3.6, `pandas.DataFrame` sorts the key order when it takes  `dict` using the function below.

```
def dict_keys_to_ordered_list(mapping):
    # when pandas drops support for Python < 3.6, this function
    # can be replaced by a simple list(mapping.keys())
    if PY36 or isinstance(mapping, OrderedDict):
        keys = list(mapping.keys())
    else:
        keys = try_sort(mapping)
    return 
```

https://github.com/pandas-dev/pandas/blob/0a082d40c4a750655a7f1ce127c58ca26cd5905e/pandas/core/common.py#L218-L225

Reply:
@harupy Love this comment! now my question all solved. Thanks for sharing your knowledge.